src/main/kotlin/no/skatteetaten/aurora/boober/service/exceptions.kt

```diff
@@ -41,6 +45,87 @@ class MultiApplicationValidationException(
     }
 }
 
+class MultiApplicationValidationResultException(
+    val valid: List<AuroraDeploymentContext> = listOf(),
+    val invalid: List<Pair<AuroraDeploymentContext?, ContextErrors?>> = listOf(),
+    val errorMessage: String = "An error occurred for one or more applications"
+) : ServiceException(errorMessage) {
+    fun toValidationErrors(): List<ApplicationError> = listOf(
+        invalid.map {
+            it.second?.let {
+                it.errors.map { t ->
+                    ApplicationError(
+                        it.command.applicationDeploymentRef.application,
+                        it.command.applicationDeploymentRef.environment,
+                        when (t) {
+                            is AuroraConfigException -> t.errors
+                            is IllegalArgumentException -> listOf(ConfigFieldErrorDetail.illegal(t.message ?: ""))
+                            else -> listOf(ErrorDetail(message = t.message ?: ""))
+                        }
+                    )
+                }
+            } ?: listOf(
+                ApplicationError(
+                    it.first!!.cmd.applicationDeploymentRef.application,
```

Comment:
Vil ikke dette føre til NPE dersom feilen er forårsaket av en exception i createAuroraDeploymentContext-kallet i createValidatedAuroraDeploymentContexts-metoden? Jeg ville jobbet mot at AuroraDeploymentContext aldri kan være null her istedenfor non-null assertion-operatoren.